From 4211e368dbcebca0eaa408461d9434b035342944 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Thu, 14 Sep 2023 16:04:10 +0200
Subject: [PATCH] Packages/Curl: Updating curl to the latest fix security:

In older version of curl 8.2.1 were several security issues.  Upgrade to the
latest version 8.3.0 for fix this security issues.  In makefile changed fix
numbering of releases to autorelease.
(Commit hash: 6fa1d817e5b1a00d7d0c8168091877476b499317)
---
 net/curl/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/net/curl/Makefile b/net/curl/Makefile
index 71f483520..c1d7abe2f 100644
--- a/net/curl/Makefile
+++ b/net/curl/Makefile
@@ -10,13 +10,13 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=curl
-PKG_VERSION:=7.83.1
-PKG_RELEASE:=$(AUTORELEASE).1
+PKG_VERSION:=8.3.0
+PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://dl.uxnr.de/mirror/curl/ \
 	https://curl.askapache.com/download/ \
 	https://curl.se/download/
-PKG_HASH:=2cb9c2356e7263a1272fd1435ef7cdebf2cd21400ec287b068396deb705c22c4
+PKG_HASH:=376d627767d6c4f05105ab6d497b0d9aba7111770dd9d995225478209c37ea63
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=COPYING
-- 
2.42.0

